src/current/cockroachcloud/connect-to-your-cluster.md

```diff
@@ -63,132 +63,77 @@ Azure Private Link is not yet available for [CockroachDB {{ site.data.products.d
 1. Run the command displayed on the **Accept VPC peering connection request** window using [Google Cloud Shell](https://cloud.google.com/shell) or using the [gcloud command-line tool](https://cloud.google.com/sdk/gcloud).
 1. On the **Networking** page, verify the connection status is **Available**.
 
-## Select a connection method
 
-1. In the top right corner of the Console, click the **Connect** button.
-
-    The **Connect** dialog displays with **IP Allowlist** selected by default.
-
-1.  Select a **Network Security** option:
-
-      You can use the **IP Allowlist** option if you have already [added an IP address to your allowlist.](#add-ip-addresses-to-the-allowlist)
+## Connect to your cluster
 
-      For AWS clusters, you can select **AWS PrivateLink** if you have already [established a PrivateLink connection](#establish-gcp-vpc-peering-or-aws-privatelink).
+1. In the top right corner of the CockroachDB {{ site.data.products.cloud }} Console, click the **Connect** button.
 
-      For GCP clusters, you can select **VPC Peering** if you have already:
-    - [Enabled VPC peering while creating your cluster]({% link cockroachcloud/create-your-cluster.md %}#step-8-enable-vpc-peering-optional)
-    - [Established a VPC Peering connection](#establish-gcp-vpc-peering-or-aws-privatelink)
+    The **Setup** page of the **Connect to cluster** dialog displays.
 
-1. From the **User** dropdown, select the SQL user you created.
-1. From the **Region** dropdown, select the region closest to where your client or application is running.
-1. From the **Database** dropdown, select the database you want to connect to.
+1. If you set up a private connection, click **AWS PrivateLink** (for clusters deployed in AWS) or **VPC Peering** (for clusters deployed in GCP) to connect privately. Otherwise, click **IP Allowlist**.
+1. Select the **SQL User**. If you have only one SQL user, it is automatically selected.
 
-    The default database is `defaultdb`. For more information, see [Default databases](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/show-databases#preloaded-databases).
+    {{site.data.alerts.callout_info}}
+    If you forget your SQL user's password, an [Org Administrator]({% link cockroachcloud/authorization.md %}#org-administrator-legacy) or a Cluster Admin on the cluster can change the password on the **SQL Users** page.
+    {{site.data.alerts.end}}
 
+1. Select the **Database**. If you have only one database, it is automatically selected.
+1. For a multiregion cluster, select the **Region** to connect to. If you have only one region, it is automatically selected.
 1. Click **Next**.
 
-1. Select a connection method (the instructions below will adjust accordingly):
+    The **Connect** page of the **Connection info** dialog displays.
+
+1. In the dialog, select the tab for a connection method, then follow the instructions below for that method.
```

Comment:
The changes in lines 66 through 86 seem largely irrelevant to the product change. Why did we make these changes here?

Reply:
@mdlinville ^^^